keycloak/themes/edu-hub/login/resources/css/stujo.css

```
@@ -0,0 +1,82 @@
+/*
+ * StuJo skin for the shared login theme. Activated via html.stujo-brand
+ * (see js/stujo-brand.js). Recreates the StuJo look: purple gradient
+ * backdrop (like the site header), white dialog card, green submit
+ * button, pink links — so the login reads as a StuJo dialog instead of
+ * the dark EduHub page.
+ */
+
+html.stujo-brand .login-pf body,
+html.stujo-brand body {
+  background-image: none !important;
+  background: linear-gradient(to right, #4b0135, #a71580, #4b0135) !important;
```

Stylelint (17.14.0)

[error] Overridden property "background-image" by shorthand "background" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)